### ISSUE TITLE
Custom metrics for precision

### ISSUE THREAD

**Issue by KDuangrattanalert** [rstudio/keras#245]
Hi,

I am doing a binary classification with Tensorflow backend. Instead of accuracy, I would like to implement custom metrics for precision. Any idea how to do this?
